filepicker/src/main/java/droidninja/filepicker/FilePickerBuilder.kt

```diff
@@ -22,6 +22,21 @@ class FilePickerBuilder {
 
     private val mPickerOptionsBundle: Bundle = Bundle()
 
+    fun setImageFileSize(fileSize: Int): FilePickerBuilder {
```

Comment:
Can you change this name to `setImageSizeLimit` and `setVideoSizeLimit`?

Reply:
Sure, I've made the changes.